homeassistant/components/nextcloud/const.py

```diff
@@ -4,3 +4,13 @@
 DOMAIN = "nextcloud"
 DEFAULT_SCAN_INTERVAL = timedelta(seconds=60)
 DEFAULT_VERIFY_SSL = True
+
+
+def isbool(val):
+    """Test if value is a Nextcloud boolean value."""
+    return isinstance(val, bool) or val in ["yes", "no"]
+
+
+def istrue(val):
+    """Test if value is a Nextcloud true value."""
+    return val is True or val in ["yes"]
```

Comment:
Those are not constants.
You should add them to another module (maybe utils.py) if you feel they are necessary.